documentation/modules/con-securing-kafka-authentication.adoc

```diff
@@ -99,16 +99,19 @@ When a `KafkaUser.spec.authentication.type` is configured with `scram-sha-512` t
 [id='assembly-kafka-broker-listener-network-policies-{context}']
 == Network policies
 
-Strimzi automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker.
-By default, a `NetworkPolicy` grants access to a listener to all applications and namespaces.
+By default, Strimzi automatically creates a `NetworkPolicy` resource for every listener that is enabled on a Kafka broker.
```

Comment:
The _by default_ sounds good when I read just the first sentence. But when I try to read the whole paragraph, it sounds a bit weird with the two _by defaults_ in the first two sentences. I'm fine if you wanna keep it ... just sharing my impression of it.

Reply:
You're right. This is why I should read the module in full after making the updates!

Removed the second "by default". Also, moved the second sentence into a separate paragraph about networkPolicyPeers.